backend/packages/Upgrade/src/api/models/Experiment.ts

```diff
@@ -182,5 +182,5 @@ export class Experiment extends BaseModel {
     nullable: true,
     type: 'json',
   })
-  public moocletDetails?: MoocletExperimentRef;
+  public MoocletExperimentRef?: MoocletExperimentRef;
```

Comment:
This should be `moocletExperimentRef`.

Reply:
this property will be going away on the Experiment model anyway